src/ScottPlot/Plottable/RadialGaugePlot.cs

```diff
@@ -0,0 +1,709 @@
+        public bool ShowGaugeValues { get; set; } = true;
+
+        public System.Drawing.Drawing2D.LineCap EndCap { get; set; } = System.Drawing.Drawing2D.LineCap.Triangle;
+
+        public System.Drawing.Drawing2D.LineCap StartCap { get; set; } = System.Drawing.Drawing2D.LineCap.Round;
+
+        /// <summary>
+        /// Controls rendering style of the concentric circles (ticks) of the web
+        /// </summary>
+        //public RadarAxis AxisType { get; set; } = RadarAxis.None;
+
+        // These 3 properties are needed as part of IPlottable
+        public bool IsVisible { get; set; } = true;
+        public int XAxisIndex { get; set; } = 0;
+        public int YAxisIndex { get; set; } = 0;
+
+        #endregion Properties & fields
+
+        /// <summary>
+        /// Initializes the plot instance.
+        /// </summary>
+        /// <param name="values">Array of values to be plotted as gauges.</param>
+        /// <param name="lineColors">Array colors for the gauges.</param>
+        public RadialGaugePlot(double[] values, Color[] lineColors)
+        {
+            GaugeColors = lineColors;
+            Update(values);
+        }
+
+        public override string ToString() =>
+            $"RadialGauge with {DataRaw.Length} points.";
+
+        /// <summary>
+        /// Replace the data values with new ones. This passed data is copied and stored in <see cref="DataRaw"/>.
+        /// Implicitly calls the <see cref="ComputeAngularData"/> routine
+        /// </summary>
+        /// <param name="values">Array of values to be plotted as gauges.</param>
+        public void Update(double[] values)
+        {
+            DataRaw = new double[values.Length];
+            Array.Copy(values, 0, DataRaw, 0, values.Length);
+
+            // Compute MaxScale and MinScale values and transform DataRaw into DataAngular
+            ComputeMaxMin();
+            ComputeAngularData();
+        }
+
+        /// <summary>
+        /// Returns the original data in case it should be needed.
+        /// </summary>
+        /// <returns>Original data vector</returns>
+        public double[] GetData() => DataRaw;
+
+        /// <summary>
+        /// Returns the computed angular data in case it should be needed.
+        /// </summary>
+        /// <returns>Angular data vector</returns>
+        public double[,] GetAngularData() => DataAngular;
+
+        /// <summary>
+        /// Converts <see cref="DataRaw"/> into <see cref="DataAngular"/>.
+        /// Depends on <see cref="DataRaw"/>, <see cref="GaugeMode"/>, <see cref="GaugeDirection"/>, <see cref="StartingAngleGauges"/>, <see cref="AngleRange"/>, and <see cref="MaxScale"/>
+        /// </summary>
+        private void ComputeAngularData()
+        {
+            // Check if there's data
+            if (DataRaw == null) return;
+            DataAngular = new double[DataRaw.Length, 2];
+
+            // Internal variables
+            float AngleSumPos = _StartingAngleGauges;
```

Comment:
Do these need to be different variables? You do the same thing to them either way.

Reply:
Yes. Although it seems they accomplish the same, they just keep the total angular count for positive and negative data values.
https://github.com/ScottPlot/ScottPlot/blob/d7f90e1346e91c60c85a45e2d2f75b36302bd19a/src/ScottPlot/Plottable/RadialGaugePlot.cs#L332-L335

Reply:
> You do the same thing to them either way

This is true. I refactored this section to be simpler:

```cs
float angleSum = StartingAngleGauges;

for (int i = 0; i < DataRaw.Length; i++)
{
    float angleSwept = (float)(AngleRange * DataRaw[i] / (MaxScale - MinScale));
    if (GaugeDirection == RadialGaugeDirection.AntiClockwise)
        angleSwept *= -1;

    double initialAngle = _GaugeMode == RadialGaugeMode.Stacked ? StartingAngleGauges : angleSum;
    angleSum += angleSwept;

    DataAngular[i, 0] = initialAngle;
    DataAngular[i, 1] = angleSwept;
}
```